app/sidekiq/evss/document_upload.rb

```diff
@@ -29,6 +35,31 @@ class EVSS::DocumentUpload
     rand(3600..3660) if count < 9
   end
 
+  sidekiq_retries_exhausted do |msg, _ex|
+    # There should be 3 args:
+    # 1) Auth headers needed to authenticate with EVSS
+    # 2) The uuid of the record in the UserAccount table
+    # 3) Document metadata
+
+    next unless Flipper.enabled?('cst_send_evidence_failure_emails')
```

Comment:
I found that using `return` actually caused an error to occur and the code in the rescue block would get executed, so using `next` to avoid that